src/scripts/main.js

```diff
@@ -1,3 +1,105 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const field = document.querySelector('.field');
+const tbody = field.querySelector('tbody');
+const minRow = 2;
+const minColumn = 2;
+const maxRow = 10;
+const maxColumn = 10;
```

Comment:
Don't need to duplicate variables with the same values
```suggestion
const minCount = 2;
const maxCount = 10;
```